gammapy/makers/safe.py

```diff
@@ -17,6 +17,11 @@
 class SafeMaskMaker(Maker):
     """Make safe data range mask for a given observation.
 
+    
+    .. warning::
+    
```

Comment:
Technically this is true for `aeff-default` `aeff-max` and `edisp-bias`as well.

Maybe :  'Note that currently some methods computing a safe energy range (`aeff-default` `aeff-max` and `edisp-bias`) determine a true energy range and apply it to reconstructed energy, effectively neglecting the energy bias.

Reply:
energy bias → energy dispersion (it's not only the mean bias that is neglected, but the error in each individual event)

Reply:
Indeed, both rms and bias play a role. And 'energy dispersion' is the term used in our IRF theory. Thanks @maxnoe